Formula/mariadb-connector-odbc.rb

```diff
@@ -1,9 +1,14 @@
 class MariadbConnectorOdbc < Formula
   desc "Database driver using the industry standard ODBC API"
   homepage "https://downloads.mariadb.org/connector-odbc/"
-  url "https://downloads.mariadb.org/f/connector-odbc-3.1.9/mariadb-connector-odbc-3.1.9-ga-src.tar.gz"
-  sha256 "5ead3f69ccde539fd7e3de6f4b8f0e6f9f6c32b4b4f082adf0e2ff110971fe1e"
-  license "LGPL-2.1"
+  url "https://mirror.netcologne.de/mariadb/connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz"
```

Comment:
maybe consistent with the original url?
```suggestion
  url "https://downloads.mariadb.org/f/connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz"
```

Reply:
looks like sha is different with mariadb.org one, `d81a35cd9c9d2e1e732b7bd9ee704eb83775ed74bcc38d6cd5d367a3fc525a34`

Reply:
I'm not sure what you're saying.  That is exactly the sha256 the PR updates us to.  What is the issue?

I just double-checked and the downloads.mariadb.org seems to always generate a 302 redirect to one if its mirror sites:
```
)$ wget https://downloads.mariadb.org/f/connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz
--2020-12-22 03:52:04--  https://downloads.mariadb.org/f/connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz
Resolving downloads.mariadb.org (downloads.mariadb.org)... 2a01:4f8:c2c:b04e::1, 116.203.207.31
Connecting to downloads.mariadb.org (downloads.mariadb.org)|2a01:4f8:c2c:b04e::1|:443... connected.
HTTP request sent, awaiting response... 302 FOUND
Location: https://mariadb.mirror.pcextreme.nl//connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz [following]
--2020-12-22 03:52:06--  https://mariadb.mirror.pcextreme.nl//connector-odbc-3.1.11/mariadb-connector-odbc-3.1.11-ga-src.tar.gz
Resolving mariadb.mirror.pcextreme.nl (mariadb.mirror.pcextreme.nl)... 2a05:1500:ff01:3:ec4:7aff:fe1e:767f, 185.107.214.114
Connecting to mariadb.mirror.pcextreme.nl (mariadb.mirror.pcextreme.nl)|2a05:1500:ff01:3:ec4:7aff:fe1e:767f|:443... connected.
HTTP request sent, awaiting response... 200 OK
```
I guess we can stick with downloads.mariadb.org if homebrew is OK with that.  I just went with netcologne.de because that's where the file came from when I downloaded it via the webbrowser via their normal flow

Reply:
BTW one issue we might hit is that the CI system seems to find downloads.mariadb.org dodgy... maybe some of the mirror sites that it redirects to aren't as good as others.  For instance, see this failure on my mariadb-connector-c PR: https://github.com/Homebrew/homebrew-core/pull/67289/checks?check_run_id=1584619895